src/SfxWeb/src/app/Common/Constants.ts

```diff
@@ -226,3 +222,29 @@ export class TelemetryEventNames {
   public static listSize = 'set list size';
   public static advancedMode = 'enable advanced mode';
 }
+
+
+export class RepairTaskMessages {
+  public static longExecutingMessage = "This update can prevent other updates from going through. Please reach out to the VMSS resource provider to figure out why the platform updates are not completing.";
```

Comment:
"This update can prevent other updates from going through. Please reach out to the Azure Compute teams (“Compute Manager/Blackbird”)       to figure out why the platform updates are not completing."